mysite/chat/views.py

```diff
@@ -161,18 +163,22 @@ def get(self, request, enroll_key, chat_id=None):
             enrolling.role = Role.ENROLLED
             enrolling.save()
 
-        chat, next_point = self.get_or_init_chat(enroll_code, chat_id)
+        chat, i_chat_id = self.get_or_init_chat(enroll_code, chat_id)
+        next_point = None
```

Comment:
No, this line must be here, cause function get_or_init_chat will init new chat only in case if chat_id is 0 (zero). Otherwise it will return exist chat or None.

Also, we need chat here to get live session progress (do we really need it to know on this page?)